packages/custom-functions-metadata/src/parseTree.ts

```diff
@@ -520,6 +536,22 @@ function getOptions(
     }
   }
 
+  if (optionsItem.requiresStreamAddress || optionsItem.requiresStreamParameterAddresses) {
+    let errorParam = optionsItem.requiresStreamAddress ? "@requiresStreamAddress" : "@requiresStreamParameterAddresses";
+
+    if (!optionsItem.stream) {
```

Comment:
Just curious, what's the difference between `optionsItem.stream` and `isStreamingFunction`?